### DISCUSSION TITLE
Which Nx version and when will angular 13 be supported ?

### DISCUSSION THREAD

**Discussion by ringodotnl** [nrwl/nx#7229] [General]
Question:
Which Nx version and when will Angular 13 (est. release date November 2021)  be supported  ?

Cheers,
Ringo.

**Comment by brandonroberts** [comment]
It depends on which version of Nx we are currently on at that time, but it will be in the Nx 13.x release as we plan to release it before Angular v13 goes final.

**Reply by ringodotnl** [reply]
@brandonroberts thank you. do you know the (est.) release date of Angular v13 ?

**Reply by PointSingularity** [reply]
@ringodotnl [Angular Releases]( https://angular.io/guide/releases) According to this it should be November 13th.
